controllers/articles.controller.js

```diff
@@ -20,4 +21,20 @@ const getArticlesById = (req, res, next) => {
     });
 };
 
-module.exports = { getArticles, getArticlesById };
+const getArticleComments = (req, res, next) => {
+  const articleId = req.params.article_id;
+  const promises = [selectArticleComments(articleId)];
+  if (articleId) {
+    promises.push(selectArticlesById(articleId));
+  }
```

Comment:
There's no need for the if statement. The only way this endpoint will be triggered is if an article id is passed in